src/main/java/com/exadel/discount/entity/User.java

```diff
@@ -35,15 +46,36 @@
             strategy = "org.hibernate.id.UUIDGenerator"
     )
     private UUID id;
+    @Column(name = "first_name", length = 50, nullable = false)
+    private String firstName;
+
+    @Column(name = "last_name", length = 50, nullable = false)
+    private String lastName;
+
+    @Column(name = "phone", length = 50, nullable = false)
+    private String phone;
 
     @Column(name = "email", length = 50, nullable = false)
     private String email;
 
+    @Column(name = "login", length = 50, nullable = false)
+    private String login;
+
     @Column(name = "password", length = 225, nullable = false)
     private String password;
 
     @Enumerated(EnumType.STRING)
     @Column(columnDefinition = "role", nullable = false)
     @Type(type = "user_role")
     private Role role;
-}
\ No newline at end of file
+
+    @OneToMany(mappedBy = "user", fetch = FetchType.LAZY, cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<Coupon> coupons = new ArrayList<>();
+
+    @OneToMany(mappedBy = "user", fetch = FetchType.LAZY, cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<Favorite> favorites = new ArrayList<>();
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name="city_id", nullable=false)
+    private City city;
```

Comment:
City not found, no import

Reply:
from the same package
package com.exadel.discount.entity;